web/src/Components/Common/Tooltip.tsx

```diff
@@ -34,9 +34,9 @@ interface TooltipProps {
 
 export const Tooltip = ({ text, children }: TooltipProps): ReactElement => {
   return (
-    <TooltipWrapper>
+    <TooltipWrapper key={text}>
       {children}
-      <EDSTooltip title={text} placement='rightTop'>
+      <EDSTooltip title={text} placement='top-end'>
```

Comment:
I got a compile error here, i think this should be: 
placement={'...''} I could not find a valid option called top-end from pycharm autocomplete... perhaps need to use "top" or something else